dvc/repo/get_url.py

```diff
@@ -5,7 +5,6 @@
 from dvc.utils import resolve_output
 
 
-@staticmethod
```

Comment:
Interesting that this didn't break it.

Reply:
It is legal, mypy just does not like it.